test/JDBC/expected/Test-sp_reset_connection-vu-verify.out

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE #babel_temp_table (ID INT identity(1,1), Data INT)
+INSERT INTO #babel_temp_table (Data) VALUES (100), (200), (300)
+GO
+~~ROW COUNT: 3~~
+
+
+SELECT * from #babel_temp_table
+GO
+~~START~~
+int#!#int
+1#!#100
+2#!#200
+3#!#300
+~~END~~
+
+
+EXEC sys.sp_reset_connection
```

Comment:
let's also add test to verify that all the active transactions are rolled back

Reply:
As pointed out to Dipesh earlier, There is an ongoing task for it which I have started working on but we do not need such tests for supporting a procedure for reset.